broker/src/test/java/io/zeebe/broker/system/partitions/ZeebePartitionTest.java

```diff
@@ -61,6 +63,41 @@ public void shouldInstallLeaderPartition() {
     verify(transition).toLeader();
   }
 
+  @Test
+  public void shouldCallOnFailureOnAddFailureListenerAndUnhealthy() {
```

Comment:
The test doesn't make it clear that the partition's health is really `Unhealthy`. Can you make that more explicit in the test?

Reply:
Done